### ISSUE TITLE
اضافه ميزه

### ISSUE THREAD

**Issue by alsophee** [PhilippC/keepass2android#851]
اريد ان أرفع مجلد كامل فيه فيه عدد من الملفات مره واحده وليس كل مره أرفق ملف

**Comment by Hustensirup** [comment]
How is this an issue?
This is more like a Feature request.

**Comment by alsophee** [comment]
I want to upload  folders  at one time to cloud  not one file and one file

**Comment by alsophee** [comment]
Why when I log on the app takes a long time to download the database Please solve the problem

**Comment by Karreg** [comment]
If the download time is long, this is more a network issue you should work on with your Internet Service Provider.
Without any context there's nothing that can be done here.
You should want to get logs, and check your database options, specifically the security options with encryption algorithm and parameters...

**Comment by alsophee** [comment]
The problem for me is that I want to choose a certain number of pictures and upload them in the cloud not every time I choose a picture and upload it this takes a lot of effort

**Comment by ouzowtf** [comment]
Putting your (porn?) pictures into a password manager in a cloud storage is not how it is intended to be used.

**Comment by alsophee** [comment]
What do you mean I have pornographic pictures?  That's not true. And also how did you know I had pornographic pictures? This is not  right i'm sure also how and how you knew my pictures did you hack  my Account. This is a criminal act. And punishable by law

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ouzowtf** [comment]
I don't know what pictures you store there, but having massive amounts of pictures in a password manager is not a good idea. Especially if it's stored in the cloud and your network connection (and maybe additionally your device) is too slow.

And if I had hacked your account I would for sure make you stop posting assumed Outlook links.
Since I did not, I only kindly ask you to stop it.

**Comment by alsophee** [comment]
I don't know exactly what you mean. Can you explain to me Exactly.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by alsophee** [comment]
I meant if I wanted to upload some private photos to the cloud or a number of pictures that's what I mean so I want to choose a number of pictures or a number of private files at once not to upload each picture individually is it possible to add this feature

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ouzowtf** [comment]
Don't use a PASSWORD manager for this!!!

And stop posting those Outlook links!!!

**Comment by Karreg** [comment]
I agree.
A password manager is not for uploading (porn?) pictures.
Stop asking, there won't be anything developed for that...

Too bad nobody except Philipp can close those issues... :(

**Comment by alsophee** [comment]
I'm very sorry because I think my retransmission program was wrong, as it sends several messages at the same time, and the problem with outlock is that it sends several messages at the same time.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by PhilippC** [comment]
while I don't think it's a high priority feature, it's still a valid feature request. It's possible to add several attachments to an entry, so why not allow to select multiple files easily?

**Comment by Karreg** [comment]
True... I think we overreacted to the multiple requests, most of them being nonsense we overlooked even the legit ones.
I don't have this use case, I only add files from Keepass2, most of them being ssh keys...

**Comment by alsophee** [comment]
If possible, I want an option to upload photos and videos without encryption to increase the speed of uploading files.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Option for multiple attachments work's in desktop. Drag drop too.

Android file manager select multiple files share with kp2a select entry to attach to?

> If possible, I want an option to upload without encryption to increase the speed of uploading files.

Wow. Than you won't need keepass. 
If encryption is to slow, go settings and change iterations to lower value.
Upload speed will be limited by bandwidth.


There is several use cases of images storing. Not only the one who mentioned porn joking.
Authentication QR codes.
Identification Scans
Bill's
Configurations

**Comment by alsophee** [comment]
I want to keep a certain key and keep the password 
Be synchronised. It's not instant, but it's in the background. So I can change a lot of things in the app.

**Comment by alsophee** [comment]
There is slowness when you open the application please fix the problem
